dvc/output/local.py

```diff
@@ -155,12 +157,7 @@ def save(self):
 
     @property
     def dir_cache(self):
-        if self.checksum not in self._dir_cache.keys():
-            self._dir_cache[
-                self.checksum
-            ] = self.repo.cache.local.load_dir_cache(self.checksum)
-
-        return self._dir_cache[self.checksum]
+        return self.repo.cache.local.load_dir_cache(self.checksum)
```

Comment:
why do we remove this optimization(?), is it not relevant for outputs in general?

Reply:
I didn't remove it, I just moved it to RemoteLOCAL class, where it is more suitable.